### PR TITLE
Minor fixes in cm3ide nodes/Source.m3

### PR DESCRIPTION
**Pull request by VictorMiasnikov**
Minor fixes in cm3ide nodes/Source.m3

### CODE REVIEW

File: m3-sys/cm3ide/src/nodes/Source.m3 [hunk 1]
```diff
@@ -23,11 +23,10 @@ VAR
   clientID := ID.Add ("clients");
 
 CONST
-  Kind = { I3, M3, IG, MG, C, CPP, H, Quake, Other };
```
Comment (jaykrell) [inline m3-sys/cm3ide/src/nodes/Source.m3:26]: Why remove this line?

File: m3-sys/cm3ide/src/nodes/Source.m3 [hunk 2]
```diff
@@ -23,11 +23,10 @@ VAR
   clientID := ID.Add ("clients");
 
 CONST
-  Kind = { I3, M3, IG, MG, C, CPP, H, Quake, Other };
   IsM3 = ARRAY Kind OF BOOLEAN { TRUE,  (* i3 *)
                                  TRUE,  (* m3 *)
                                  TRUE,  (* ig *)
-                                 TRUE,  (* mg )
+                                 TRUE,  (* mg *)
```
Comment (jaykrell) [inline m3-sys/cm3ide/src/nodes/Source.m3:29]: Oops, thank you. I am not sure I needed to change cm3ide.
Maybe revert, just around here?

Comment (VictorMiasnikov) [reply m3-sys/cm3ide/src/nodes/Source.m3:29]: } Oops, thank you
 No problem -)
 
} I am not sure I needed to change cm3ide.
} Maybe revert, just around here?
 I do not shue.
 In any case "my minor fixes" are need.

**Comment by VictorMiasnikov** [comment]
} Jaykrell merged commit 2969499 into modula3:master
Thanks!

**Comment by VictorMiasnikov** [comment]
Because this line exist in .i3 file 28.07.2021, 12:12, "Jay Krell" ***@***.******@***.*** commented on this pull request.In m3-sys/cm3ide/src/nodes/Source.m3:> @@ -23,11 +23,10 @@ VAR
   clientID := ID.Add ("clients");
 
 CONST
-  Kind = { I3, M3, IG, MG, C, CPP, H, Quake, Other };
Why remove this line?—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Pull request merged**